tasks/stylus.js

```diff
@@ -81,18 +81,20 @@ module.exports = function(grunt) {
     } catch (e) {}
 
     s.render(function(err, css) {
+      var map = s.sourcemap ? JSON.stringify(s.sourcemap) : null;
```

Comment:
Changes in this file come basically from PR https://github.com/gruntjs/grunt-contrib-stylus/pull/121 by @pixeldrew